### PR TITLE
UI focus feature fixed

### PR DESCRIPTION
**Pull request by shubhay09**

### CODE REVIEW

**Comment by elliotcourant** [comment]
Why close the original pull request?

**Comment by shubhay09** [comment]
i thought it was not going to work can you review the changes ??

**Comment by shubhay09** [comment]
i dont know why im getting deepsource javascript error can you help out ?

**Review by shubhay09 — COMMENTED** [review]
review required

File: ui/components/Authentication/Registration/RegisterView.tsx [hunk 1]
```diff
@@ -296,20 +296,20 @@ export default function RegisterView(): JSX.Element {
                         }
                         label={
                           <Fragment>
-                            I agree to monetr's&nbsp;
+                             I agree to monetr's&nbsp;
                             <a
-                              className="hover:text-blue-500 hover:underline"
+                              className="hover:text-blue-500 hover:underline focus:ring-2 focus:ring-blue-600"
```
Comment (elliotcourant) [inline ui/components/Authentication/Registration/RegisterView.tsx:301]:
```suggestion
                              className="hover:text-blue-500 hover:underline focus:ring-2 focus:ring-blue-500 focus:underline"
```

File: ui/components/Authentication/Registration/RegisterView.tsx [hunk 2]
```diff
@@ -296,20 +296,20 @@ export default function RegisterView(): JSX.Element {
                         }
                         label={
                           <Fragment>
-                            I agree to monetr's&nbsp;
+                             I agree to monetr's&nbsp;
```
Comment (elliotcourant) [inline ui/components/Authentication/Registration/RegisterView.tsx:299]:
```suggestion
                            I agree to monetr's&nbsp;
```

File: ui/components/Authentication/Registration/RegisterView.tsx [hunk 3]
```diff
@@ -296,20 +296,20 @@ export default function RegisterView(): JSX.Element {
...
                             <a
-                              className="hover:text-blue-500 hover:underline"
+                              className="hover:text-blue-500 hover:underline focus:ring-2 focus:ring-blue-600"
                               href='https://github.com/monetr/legal/blob/main/TERMS_OF_USE.md'>
                               Terms of Use
                             </a> and&nbsp;
                             <a
-                              className="hover:text-blue-500 hover:underline"
+                              className="hover:text-blue-500 hover:underline focus:ring-2 focus:ring-blue-600"
```
Comment (elliotcourant) [inline ui/components/Authentication/Registration/RegisterView.tsx:306]:
```suggestion
                              className="hover:text-blue-500 hover:underline focus:ring-2 focus:ring-blue-500 focus:underline"
```

File: ui/components/Authentication/Registration/RegisterView.tsx [hunk 4]
```diff
@@ -296,20 +296,20 @@ export default function RegisterView(): JSX.Element {
...
                               href='https://github.com/monetr/legal/blob/main/PRIVACY.md'
                             >
                               Privacy Policy
                             </a>
                           </Fragment>
-                        }
+                         }
```
Comment (elliotcourant) [inline ui/components/Authentication/Registration/RegisterView.tsx:312]:
```suggestion
                        }
```

**Review by elliotcourant — CHANGES_REQUESTED** [review]
Just some small tweaks then this should be good to go!

**Comment by shubhay09** [comment]
changes have been made

**Review by shubhay09 — COMMENTED** [review]
changes have been you can review now and let me know anything else to be done or changed

**Pull request merged**